frontend/util.lua

```diff
@@ -652,12 +653,12 @@ function util.getFriendlySize(size)
     size = tonumber(size)
     if not size or type(size) ~= "number" then return end
     local s
-    if size > 1024*1024*1024 then
-        s = string.format("%4.1f GB", size/1024/1024/1024)
-    elseif size > 1024*1024 then
-        s = string.format("%4.1f MB", size/1024/1024)
+    if size > 1073741824 then
```

Comment:
From the luajit notes posted at https://github.com/koreader/koreader/pull/5625#issuecomment-557425843
> the constant expression is folded

So, it shouldn't hurt letting it be `1024*1024*1024` - but if you don't trust it :) at least, put that as a comment so we don't have to wonder what that magic number `1073741824` is :)
`if size > 1073741824 then -- 1024*1024*1024`